source/models/index.py

```diff
@@ -0,0 +1,42 @@
+import os
+from outputs.exceptions import *
+
+class Index:
+    def __init__(self, table, index_value):
+        self.table = table
+        self.index_value = index_value
+
+    def serialize(self):
+        os.makedirs(self.get_path(), exist_ok = True)
+    
+    def get_path(self):
+        return os.path.join(self.table.get_path(), os.path.join("indices" , self.index_value))
```

Comment:
why don't u have `self.__path = os.path.join(self.table.get_path(), os.path.join("indices" , self.index_value))`

Reply:
because i had one ```get path``` function I though i should not make another get path but you are right.
i did it